fabricexample/src/App.tsx

```diff
@@ -115,6 +114,8 @@ export default class App extends React.Component<{}, State> {
           onPageSelected={this.onPageSelected}
           onPageScrollStateChanged={this.onPageScrollStateChanged}
           pageMargin={10}
+          layoutDirection="ltr"
+          overdrag={false}
```

Comment:
We should make it as false inside library

Reply:
It's false by default. I can remove it from here as it makes no effect